test/integration/sequelize.test.js

```diff
@@ -249,6 +249,18 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
       return this.sequelize.query(this.insertQuery);
     });
 
+    it('executes a query with benchmarking option', function() {
```

Comment:
Could you add a test that tests the global option as well? You can use `support.createSequelizeInstance` to get a new instance with benchmarking enabled